### PR TITLE
Add script to calculate percentage of translation

### PR DESCRIPTION
**Pull request by kbariotis**
Adds a GH action that will calculate the completion percentage for each translation file.

Based on that, we will selectively show only the languages that fulfill a threshold.

https://github.com/excalidraw/excalidraw/issues/1823

Automation will be triggered on master after a translation PR is merged.

Example of automated commit https://github.com/excalidraw/excalidraw/pull/1826/commits/7846b1b6f66fdd62993012c4a9e07069f8820f0b

### CODE REVIEW

**Comment by dwelle** [comment]
I think we're good to go, if we want to keep it at 85, that is. So let's remove the test translation and ship.

File: src/locales/percentages.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"en":100,"ar-SA":57,"bg-BG":81,"ca-ES":92,"de-DE":99,"el-GR":98,"es-ES":97,"fa-IR":99,"fi-FI":100,"fr-FR":99,"he-IL":94,"hi-IN":99,"hu-HU":58,"id-ID":59,"it-IT":97,"ja-JP":77,"ko-KR":72,"nb-NO":99,"nl-NL":85,"pl-PL":99,"pt-PT":100,"ru-RU":81,"sq-AL":42,"tr-TR":98,"uk-UA":99,"zh-CN":99,"zh-TW":99}
```
Comment (lipis) [inline src/locales/percentages.json:1]: Can we pretty print that?

File: .eslintignore [hunk 2]
```diff
@@ -3,3 +3,4 @@ build/
 package-lock.json
 .vscode/
 firebase/
+src/locales/percentages.json
```
Comment (lipis) [inline .eslintignore:6]: If we pretty print it there is no need to if ignore it

File: scripts/build-locales-coverage.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    file !== "build-percentages.js" &&
+    file !== "en.json" &&
+    file !== "percentages.json",
+);
+
+const percentages = {
+  en: 100,
+};
+
+for (let i = 0; i < locales.length; i++) {
```
Comment (lipis) [inline scripts/build-locales-coverage.js:22]: I prefer `index` instead of a single letter var

**Comment by lipis** [comment]
I guess it only commits when there is new diff.. right?

File: .github/workflows/locales-coverage.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: Create report file
+        run: |
+          node scripts/build-locales-coverage.js
```
Comment (lipis) [inline .github/workflows/locales-coverage.yml:23]: Let's add a new script in the `package.json` and run it through there, so we can also test/run it easier locally.

File: src/i18n.ts [hunk 5]
```diff
@@ -27,6 +32,10 @@ export const languages = [
   { lng: "he-IL", label: "עברית", data: "he-IL.json", rtl: true },
 ];
 
+export const languages = allLanguages.filter(
+  (lang) => percentages[lang.lng] > 85,
+);
```
Comment (lipis) [inline src/i18n.ts:37]: Could we also sort them automatically by label and leave English first?

File: src/locales/percentages.json (hunk 1 above)
Comment (kbariotis) [reply src/locales/percentages.json:1]: Nop, it's auto generated.

File: src/locales/percentages.json [hunk 6]
```diff
@@ -0,0 +1 @@
+{"ar-SA":57,"bg-BG":81,"ca-ES":92,"de-DE":99,"el-GR":96,"en":100,"es-ES":97,"fa-IR":99,"fi-FI":100,"fr-FR":99,"he-IL":94,"hi-IN":99,"hu-HU":58,"id-ID":59,"it-IT":97,"ja-JP":77,"ko-KR":72,"nb-NO":99,"nl-NL":85,"pl-PL":99,"pt-PT":100,"ru-RU":81,"sq-AL":42,"tr-TR":98,"uk-UA":99,"zh-CN":99,"zh-TW":99}
```
Comment (lipis) [inline src/locales/percentages.json:1]: Can we pretty print this?

Comment (kbariotis) [reply src/locales/percentages.json:1]: Nop, it's auto generated

File: package.json [hunk 7]
```diff
@@ -72,24 +72,25 @@
   },
```
Comment (kbariotis) [inline package.json:72]: Changes are sorting mostly.

**Pull request merged**